src/server/pegasus_server_impl.cpp

```diff
@@ -46,6 +46,7 @@ namespace pegasus {
 namespace server {
 
 DEFINE_TASK_CODE(LPC_PEGASUS_SERVER_DELAY, TASK_PRIORITY_COMMON, ::dsn::THREAD_POOL_DEFAULT)
+DSN_DECLARE_int32(read_amp_bytes_per_bit);
```

Comment:
I suggest this config mutable.

Reply:
it actually pass to rocksdb, so won't be updated rocksdb though set mutable